src/components/Quote.jsx

```diff
@@ -0,0 +1,29 @@
+import { React, useState, useEffect } from 'react';
+
+function Quote() {
+  const [data, setData] = useState([]);
+  useEffect(() => {
+    fetch('https://api.api-ninjas.com/v1/quotes', {
+      headers: {
+        'X-Api-Key': 'OpjST6lLVPMztqEXqvVpUQ==0mUc3zSnJ67pxMdw',
+      },
+    })
+      .then((response) => response.json())
+      .then((apidata) => setData(apidata[0]));
+  }, []);
```

Comment:
- Please check the requirements you should: 
`Show a loading state
Show an error state`


- [ ] It would be better when fetching data you check if the data is still loading or if there is an error, Kindly check if your data is loading and if it has fetched you then display data, also if the fetch is not successful you should also display an error message👍🏿